static/css/openfga.css

```diff
@@ -486,3 +488,32 @@ pre {
   background-color: var(--ofga-neutral-black);
   font-size: 1rem;
 }
+
+/* Prism - DSL */
+.language-dsl .token.fga-type,
+.language-dsl .token.fga-definition {
+  color: var(--ofga-color-primary);
+}
+
+.language-dsl .token.plain {
+  color: var(--ofga-color-secondary-light);
+}
+
+.language-dsl .token.fga-self {
+  color: var(--ofga-color-secondary);
+}
+
+.language-dsl .token.fga-keyword,
+.language-dsl .token.fga-operator,
+.language-dsl .token.fga-tuple-to-userset {
+  /* color: rgb(255, 68, 221); */
+  /* color: var(--ofga-neutral-base); */
```

Comment:
Nitpick: We should clean up these commented out lines just to avoid the debt of having to fix them later.

Reply:
done!